zerver/lib/send_email.py

```diff
@@ -28,8 +32,15 @@ def build_email(template_prefix, to_email, from_email=None, reply_to_email=None,
     message = loader.render_to_string(template_prefix + '.txt',
                                       context=context, using='Jinja2_plaintext')
     html_message = loader.render_to_string(template_prefix + '.html', context)
-    if from_email is None:
-        from_email = settings.NOREPLY_EMAIL_ADDRESS
+
+    # We check for from_name == "" becacuse in forms.py we overwrite the Django send_mail function
```

Comment:
This is no longer true, right?
Would this allow us to put the default values in the argument list itself? (Not a big deal if not, but it is a little clearer if so.)